### PR TITLE
modify makefile to make languages dir persistent

### PR DESCRIPTION
**Pull request by abulka**
Later versions of Wordpress store translations in `wp-content/languages`.  I believe this directory should be made persistent (like plugins and uploads are) so that translation upgrades via the wordpress admin UI can easily be made.

Copying translations into a local repo and pushing them up is not a good alternative since the files and directory structures of that directory are nested and complex 

.. |   |   |  
-- | -- | -- | --
plugins |   |   |  
themes |   |   |  
admin-en_AU.mo | 374.03 Kb | 2018-Oct-29 |  
admin-en_AU.po | 518.38 Kb | 2018-Oct-29 |  
admin-network-en_AU.mo | 41.71 Kb | 2018-Oct-29 |  
admin-network-en_AU.po | 56.81 Kb | 2018-Oct-29 |  
en_AU.mo | 238.07 Kb | 2018-Oct-29 |  
en_AU.po | 414.71 Kb | 2018-Oct-29 |

and change when new plugins are added and translations upgraded.  There is no easy way to determine what files to put into the local repo except to apply upgrades on the server, note the differences and copy them down to the local repo - which is laborious and error prone.

A simple change to the makefile to make this directory be persistent is a much cleaner and user friendly solution.  I've tried this change and it works. Note that the need for language upgrades only kicks in when you switch wordpress to a non US language.

### CODE REVIEW

**Comment by josegonzalez** [comment]
What happens for existing repos?

**Comment by abulka** [comment]
I'm not a wordpress expert but my observation is that the wp-content/languages directory only gets created and populated if you click the "Update Translations" button underneath "New translations are available" within the "Updates" section of the Wordpress admin UI.

For existing deployments: You won't have ever run the new proposed dokku-wordpress Makefile, so the proposed new persistence mapping of the wp-content/languages directory does not exist.  Clicking the "Update Translations" button will create and populate the wp-content/languages directory, but that directory will not survive an app restart/server reboot. This means that eventually the update icon will light up again and you will have to press the "Update Translations" button again.  This is the annoying issue this pull request is trying to solve.

The typical user with an existing deployment, who has not bothered creating a wp-content/languages directory in their local repo (most people) will therefore probaby dearly want to run the 3 newly proposed lines of code to create the new persistence mapping and upgrade their deployment, before hitting the  "Update Translations" button.  Viz:

    mkdir -p /var/lib/dokku/data/storage/$APP_NAME-languages
    chown 32767:32767 /var/lib/dokku/data/storage/$APP_NAME-languages
    dokku storage:mount $(APP_NAME) /var/lib/dokku/data/storage/$APP_NAME-languages:/app/wp-content/languages

For new deployments, the above commands are already in the proposed new Makefile.

Advanced: For an existing deployment where you have taken the unusual step of creating a wp-content/languages directory in your local repo and populated it with translation files, then you are probably in full control and know what you are doing, and persistence mapping of the wp-content/languages directory is not wanted and need not be performed.  In fact that tribe of people who want to control everything via their local repos, would similarly have needed to take the initiative and have omitted the existing Makefile recommended commands that involve the persistence mapping of the wp-content/plugins directory.  This is because persistence mappings of remote directories block git pushes to those directories.

To summarise:
- Old deployment: should run 3 lines of code above and press the update translations button
- New deployments: press the update translations button
- Old deployments that are managing their wp-content/languages directory via local repo - do nothing
- New deployments that want to manage their wp-content/languages directory via local repo - omit executing the 3 lines of code above when prompted by the Makefile.  Incidentally, with the current dokku-wordpress Makefile, any new deployments who want to manage their plugins via local repo will similarly need to skip the mapping of the wp-content/plugins directory - so this need to skip various persistence mappings recommended by the Makefile is nothing new.
